scripts/linux/nv/gitlab_build_valgrind.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+mkdir -p build && cd build
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -OLP_SDK_BUILD_EXAMPLES=ON -DOLP_SDK_BUILD_DOC=ON -DBUILD_SHARED_LIBS=ON ..
```

Comment:
```suggestion
cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DOLP_SDK_BUILD_EXAMPLES=ON -DOLP_SDK_BUILD_DOC=ON -DBUILD_SHARED_LIBS=ON ..
```

What about tests? Do you need to build tests for this?
If you need then better explicitly declare that with -DOLP_SDK_ENABLE_TESTING=ON.
If you do not need to build tests then disable them with -DOLP_SDK_ENABLE_TESTING=OFF

Reply:
Done.Thanks!
Added -DOLP_SDK_ENABLE_TESTING=ON because we need them.